source/funkin/play/PlayState.hx

```diff
@@ -1335,64 +1335,13 @@ class PlayState extends MusicBeatSubState
   #end
 
   /**
-   * Removes any references to the current stage, then clears the stage cache,
-   * then reloads all the stages.
-   *
-   * This is useful for when you want to edit a stage without reloading the whole game.
-   * Reloading works on both the JSON and the HXC, if applicable.
-   *
    * Call this by pressing F5 on a debug build.
    */
-  override function debug_refreshModules():Void
+  override function reloadAssets():Void
   {
-    // Prevent further gameplay updates, which will try to reference dead objects.
-    criticalFailure = true;
-
-    // Remove the current stage. If the stage gets deleted while it's still in use,
-    // it'll probably crash the game or something.
-    if (this.currentStage != null)
-    {
-      remove(currentStage);
-      var event:ScriptEvent = new ScriptEvent(DESTROY, false);
-      ScriptEventDispatcher.callEvent(currentStage, event);
```

Comment:
Was going to say this is important, but calling `loadPlayState` calls `switchState`m which calls `destroy`m which calls `performCleanup`, and that basically does all this anyway.

Reply:
i removed all of that code because wasn't even used, the plugin had more priority